highline.gemspec

```diff
@@ -31,5 +31,6 @@ minutes of work.
 END_DESC
 
   spec.add_development_dependency "code_statistics"
+  spec.add_development_dependency "m"
```

Comment:
@matrinox I think we can live with this "development" dependency as it does not affect negative most people. But affect positively you and other developers.